extension/LoadDescriptors.ps1

```diff
@@ -0,0 +1,21 @@
+# Requires the bulk load utility from the Ed-Fi-ODS repository
+$args = @(
+    # Uncomment the line below and set your year if you are using the
+    # year-specific mode in the ODS / API.
+    # "--year", "2021"
+    "--retries", 0,
+    "--data",  ".\descriptors",
+    "--baseUrl", "http://localhost:54746/",
+    "--working", "C:\temp\bulkd-upload-client",
+    # Replace with valid key and secrete from your installation
+    "--key", "5RUEXTjlDL8HebpyY2Gzg",
+    "--secret", "HB3pt2Y9D4MCG0AmRLPrc",
```

Comment:
In development, these are randomly-generated values and saving them in source control is harmless... unless someone puts real production credentials in here. I should probably add a warning about that.

Reply:
done